deployment/docker_compose/docker-compose.dev.yaml

```diff
@@ -99,10 +99,10 @@ services:
       retries: 3
       start_period: 3s
 
-  object-storage:
+  object_storage:
```

Comment:
Switch to this might cause Windows or MacOS not able to connect to those services in docker compose

Reply:
https://github.com/TinhHoaSolutions-EzHR/chatbot/pull/127/commits/4c9a3024bc726219fa40d91465ba15d3b0388824